module #3. REST API Advanced/src/main/resources/application-dev.yml

```diff
@@ -0,0 +1,20 @@
+spring:
+  config:
+    activate:
+      on-profile: dev
+  datasource:
+    url: jdbc:postgresql://localhost:5432/gifts_dev
+    username: lobur
+    password: pass
+    driver-class-name: org.postgresql.Driver
+    hikari:
+      maximum-pool-size: 10
+  jpa:
+    hibernate:
+      ddl-auto: update
```

Comment:
Since you use lombok would be btter to use "validate" here instead of "update" so hibernate wont manage database status